modules/EvseManager/EvseManager.cpp

```diff
@@ -1395,7 +1395,7 @@ void EvseManager::cable_check() {
         float ev_max_voltage = 500.;
 
         if (ev_info.maximum_voltage_limit.has_value()) {
-            EVLOG_info << "EV reports " << ev_info.maximum_voltage_limit.has_value() << " V as maximum voltage";
+            EVLOG_info << "EV reports " << ev_info.maximum_voltage_limit.value() << " V as maximum voltage";
             ev_max_voltage = ev_info.maximum_voltage_limit.value();
         } else {
             EVLOG_error << "CableCheck: Did not receive EV maximum voltage, falling back to 500V";
```

Comment:
I would probably add a constant for the 500V, like `DEFAULT_MAXIMUM_VOLTAGE`.  Otherwise, if this gets changed, chances are likely that the `EVLOG_error` still logs 500V.  One may also use
`const auto ev_max_voltage = ev_info.maximum_voltage_limit.value_or(DEFAULT_MAXIMUM_VOLTAGE)` then.